jdaviz/configs/default/plugins/model_fitting/tests/test_plugin.py

```diff
@@ -155,6 +155,7 @@ def test_register_cube_model(cubeviz_helper, spectrum1d_cube):
     assert test_label in cubeviz_helper.app.data_collection
 
 
+@pytest.mark.skip()
```

Comment:
Why skipped? Please provide `reason=`, see https://docs.pytest.org/en/stable/how-to/skipping.html#skipping-test-functions

Reply:
fixed in 3156 branch